### PR TITLE
User guide French Completion

### PR DESCRIPTION
**Pull request by el00ruobuob**
This includes the advanced binary verification guide in French, along with typos correction on other user-guides.

### CODE REVIEW

**Comment by el00ruobuob** [comment]
Tested ok on the following platforms:

  | Edge | IE 11 | Chrome | Firefox | Opera | Vivaldi
-- | -- | -- | -- | -- | -- | --
Windows 7 | - | OK | OK | OK | OK | OK
Windows 10 | OK | OK | OK | OK | OK | OK
Ubuntu 18.04 | - | - | OK | OK | OK | OK
CentOS 7 | - | - | - | OK | OK | OK

**Comment by el00ruobuob** [comment]
Ready for review @erciccione @assylias

File: _i18n/fr/resources/user-guides/restore_account.md [hunk 1]
```diff
@@ -14,11 +14,11 @@
...
 
-Après avoir saisit les 25 mots de la phrase mnamonique et avoir choisit la hauteur de bloc, monero-wallet-cli va généré la même adresse publique et clef d'audit que pour votre ancien portefeuille et commancera le processus de réfraichissement automatiquement. (Soyez patient car le processus de rafraichissement peut être long)
+Après avoir saisit les 25 mots de la phrase mnémonique et avoir choisit la hauteur de bloc, monero-wallet-cli va généré la même adresse publique et clef d'audit que pour votre ancien portefeuille et commencera le processus de rafraichissement automatiquement. (Soyez patient car le processus de rafraichissement peut être long)
```
Comment (assylias) [inline _i18n/fr/resources/user-guides/restore_account.md]: rafraîchissement (accent circonflexe)

File: _i18n/fr/resources/user-guides/solo_mine_GUI.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ Il est très facile de miner en solo avec la GUI officielle. Si vous ne l'avez p
 
 <img src="png/solo_mine_GUI/01.PNG" style="width: 600px;"/>
 
-Cliquez sur l'onglet "Anvancé". Vous devriez voir aparaitre d'autres options. Cliquez maintenant sur le sous-onglet "Mine".
+Cliquez sur l'onglet "Avancé". Vous devriez voir apparaitre d'autres options. Cliquez maintenant sur le sous-onglet "Mine".
```
Comment (assylias) [inline _i18n/fr/resources/user-guides/solo_mine_GUI.md]: apparaître (accent circonflexe)

File: _i18n/fr/resources/user-guides/verification-allos-advanced.md [hunk 3]
```diff
@@ -1,79 +1,80 @@
...
-### 2.1. Get Signing Key
+## 2. Vérifier et Importer la Clef de Signature
 
-On Windows or Mac, go to [Fluffypony's GPG key](https://raw.githubusercontent.com/monero-project/monero/master/utils/gpg_keys/fluffypony.asc), which he uses to sign the Monero binaries, and save the page as `fluffypony.asc` to your home directory.
+Cette rubrique couvre le téléchargement de la clef de signature Monero, la vérification que cette clef est correcte et l'import de la clef dans GnuPG.
 
-On Linux, you can download Fluffypony's signing key by issuing the following command:
+### 2.1. Obtenir la Clef de Signature
+
+Sut Windows et Mac, rendez-vous sur [la clef GPG de Fluffypony](https://raw.githubusercontent.com/monero-project/monero/master/utils/gpg_keys/fluffypony.asc), qu'il utilise pour signer les binaires Monero et sauvegardez la page sous `fluffypony.asc` dans votre répertoire utilisateur.
```
Comment (assylias) [inline _i18n/fr/resources/user-guides/verification-allos-advanced.md]: Sut -> Sur

File: _i18n/fr/resources/user-guides/verification-allos-advanced.md [hunk 4]
```diff
@@ -1,79 +1,80 @@
...
-On all operating systems, check the fingerprint of `fluffypony.asc` by issuing the following command in a terminal:
+Sur l'ensemble des systèmes d'exploitation, vérifiez l'empreinte de `fluffypony.asc` en tapant la commande suivante dans un terminal :
 
 ```
 gpg --keyid-format long --with-fingerprint fluffypony.asc
 ```
 
 
-Verify the fingerprint matches:
+Vérifiez que l'empreinte bien :
```
Comment (assylias) [inline _i18n/fr/resources/user-guides/verification-allos-advanced.md]: **est** bien

File: _i18n/fr/resources/user-guides/restore_account.md (hunk 1 above)
Comment (el00ruobuob) [reply _i18n/fr/resources/user-guides/restore_account.md]: ok

File: _i18n/fr/resources/user-guides/solo_mine_GUI.md (hunk 2 above)
Comment (el00ruobuob) [reply _i18n/fr/resources/user-guides/solo_mine_GUI.md]: ok

File: _i18n/fr/resources/user-guides/verification-allos-advanced.md (hunk 3 above)
Comment (el00ruobuob) [reply _i18n/fr/resources/user-guides/verification-allos-advanced.md]: ok

File: _i18n/fr/resources/user-guides/verification-allos-advanced.md (hunk 4 above)
Comment (el00ruobuob) [reply _i18n/fr/resources/user-guides/verification-allos-advanced.md]: ok

**Comment by el00ruobuob** [comment]
Thanks @assylias. Corrections applied.

**Comment by el00ruobuob** [comment]
Rebased.

**Pull request merged**